gdal/doc/source/drivers/vector/lvbag.rst

```diff
@@ -23,8 +23,6 @@ field according to the BAG 2.0 specfication.
 
 More information about the LV BAG 2.0 can be found at https://zakelijk.kadaster.nl/bag-2.0
 
-LV BAG model definitions are available at https://zakelijk.kadaster.nl/documents/20838/87954/XSD%27s+BAG+2.0+Extract
-
```

Comment:
If this link is giving 404, is there any chance to update the link instead of removing it?